docs/_documentations/mdt-eclipse-usingadifferenttemplate.md

```diff
@@ -14,9 +14,39 @@ parent: usingcodewind
 
 When you create a new project, you can choose from the default set of templates available, or you can choose from a template source of your own. Use Codewind to manage templates. Templates can make application development easier by providing a structure and boilerplate code to help get a new project started.
 
-- To open the Template Source Manager, right-click **Projects (Local)**.
-- Click **Manage Template Sources** to add a source to the table.
-- You can perform the following actions in the dialog window:
-  - Use the check boxes to enable or disable sources.
-  - Use **Add** to add a new source.
-  - Use **Remove** to remove non-default sources.
\ No newline at end of file
+- To open the Template Source Manager, right-click **Projects (Local)**, and then click **Manage Template Sources**. The **Template Source Manager** appears. 
+- You can perform the following actions in the **Template Source Manager**:
+  - Add a new template source to the table.
+  - Remove non-default sources.
+  - Enable a source to make that source template appear in the **Create Project** wizard.
+  - Disable a source to remove that source template from the **Create Project** wizard.
+
+## Adding your own templates
```

Comment:
Should this information be shared somehow instead of duplicated?  When IntelliJ comes along it will be duplicated 3 times.  A link to a shared file?

Reply:
The current IDE-based help structure dictates we duplicate content across the piece. We're currently looking at user journeys and changing the landing pages accordingly, this for a future iteration.